pom-dependency-tree.txt

```diff
@@ -1,4 +1,4 @@
-ai.elimu:webapp:war:2.5.98-SNAPSHOT
+ai.elimu:webapp:war:2.5.99-SNAPSHOT
```

Comment:
@jo-elimu Do we want to use a snapshot release here?

Reply:
@tuancoltech Good question. The challenge with this file is that it is only generated when the application is launched (during `localhost` development). So it also often happens that code changes are made without this file being updated. It's not a perfect setup, but I think it's better than not keeping track of the dependency tree at all.